addons/web/static/tests/core/popover/popover_tests.js

```diff
@@ -264,3 +264,37 @@ QUnit.test("within iframe", async (assert) => {
     assert.strictEqual(popoverBox.top, expectedTop);
     assert.strictEqual(popoverBox.left, expectedLeft);
 });
+
+QUnit.test("popover fixed position", async (assert) => {
+    const container = document.createElement("div");
+    container.id = "container";
+    container.style.backgroundColor = "pink";
+    container.style.height = "450px";
+    container.style.width = "450px";
+    container.style.display = "flex";
+    container.style.alignItems = "center";
+    container.style.justifyContent = "center";
+    popoverTarget.style.backgroundColor = "yellow";
```

Comment:
Do we need to go that far into popover specs? :smile:

Reply:
for the background no, for ffor the display flex yes :) (I copy / pasted from a test above)